src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/uploadfurtherevidence/UploadFurtherEvidenceAboutToSubmitHandler.java

```diff
@@ -103,6 +134,10 @@ private boolean isFileAPdf(DraftSscsDocument doc) {
                 && PDF_FILE_TYPE.equalsIgnoreCase(getExtension(doc.getValue().getDocumentLink().getDocumentFilename()));
     }
 
+    private boolean hasMp3OrMp4(List<DraftSscsDocument> draftSscsFurtherEvidenceDocument) {
+        return ofNullable(draftSscsFurtherEvidenceDocument).orElse(emptyList()).stream().anyMatch(doc -> MEDIA_FILE_TYPES.contains(lowerCase(getExtension(doc.getValue().getDocumentLink().getDocumentFilename()))));
+    }
```

Comment:
I added below method to DocumentUtil class. As this has to checked in meny different places, it would be good to in one place. It already have a method to check a file is a pdf.
 public static boolean isFileAMedia(DocumentLink docLink) {
        return docLink != null
                && isNotBlank(docLink.getDocumentUrl())
                && equalsAnyIgnoreCase(getExtension(docLink.getDocumentFilename()), "mp3", "mp4");
    }